### DISCUSSION TITLE
Is there a way to show sum a column at [Aug 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3363] [Discord Threads]
> 🙋‍♂️ **AlexU**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@706382317803012167> Not possible  
#  
> 🙋‍♂️ **AlexU**  
#### But maybe in the future?  
#  
> 💁 **o1lab**  
#### ya sure, we can think about it  
#  
> 💁 **o1lab**  
#### would you like to have some other function also possible to be displayed - like average and so on ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1009606287446777906)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
